.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,54 @@
+name: "CI Tests"
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container: elixir:1.10-alpine
+    services:
+      db:
+        image: postgres:12.3
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.head_ref }}
+
+    - name: Update container
+      run: |
+        apk update
+        apk add build-base
+        mix local.rebar --force
+        mix local.hex --force
+    - name: Install Dependencies
+      run: mix deps.get
+
+    - name: Create database
+      run: mix ecto.create
+      env:
+        MIX_ENV: test
+        DB_HOST: db
+
+    - name: Migrate database
```

Comment:
FYI we updated the Github action workflow on Omise Labs to use https://github.com/actions/cache to cache the build app to speed up tests. We will include it in the Elixir template during the Growth project.

Reply:
Wow, let see how it works then, never work with Github action much, once that merged to Elixir template, I will adjust on this PR